message/validation/consensus_validation.go

```diff
@@ -96,8 +100,8 @@ func (mv *messageValidator) validateConsensusMessageSemantics(
 	}
 
 	if len(signedSSVMessage.FullData) != 0 {
-		if consensusMessage.MsgType == specqbft.PrepareMsgType ||
-			consensusMessage.MsgType == specqbft.CommitMsgType && len(signedSSVMessage.GetOperatorIDs()) == 1 {
+		// Rule: Prepare or commit messages must not have full data
+		if msgType == specqbft.PrepareMsgType || msgType == specqbft.CommitMsgType && len(signers) == 1 {
```

Comment:
I rather write conditions like this with parentheses as it makes things clearer
`if msgType == specqbft.PrepareMsgType || (msgType == specqbft.CommitMsgType && len(signers) == 1) {`

Reply:
@y0sher I don't mind, done